lib/store.js

```diff
@@ -1,9 +1,11 @@
 import { configureStore } from "@reduxjs/toolkit";
 import apiReducer from "./features/apiSlice";
+import subscriptionStatusReducer from "./features/subscriptionSlice";
 export const makeStore = () => {
   return configureStore({
     reducer: {
       api: apiReducer,
+      subscription: subscriptionStatusReducer,
```

Comment:
Minor, but would 'subscription status' be a better naming?

Reply:
Agreed, I'll change it to `subscriptionStatus`.